Pipfile

```diff
@@ -7,13 +7,13 @@ verify_ssl = true
 bandit = "~=1.7"
 black = "~=22.8"
 click = "~=8.1"
+decorator = "~=5.1"
 isort = "~=5.10.0"
 mypy = "~=0.950"
+panther-analysis-tool = "*"
```

Comment:
I believe moving panther-analysis-tool to the dev section means that people would only have panther_analysis_tool installed if they ran `pipenv install --dev` .. That is the pipenv install command that happens when you run `make install`, though it's potentially not what some python ecosystem folks might do. We might break some users if they aren't using the `make` command to get going. 

I'm not saying that we shouldn't ship this because it might break users who aren't explicitly following directions, and I believe we should make sure that we're ready with CustomerSupport to respond with a fix for folks that this might break.

Reply:
Gotcha, so you are saying most folks run a `pipenv install` manually today? (and not have `pat` installed locally on their machine).  I can give CS a heads up! We want to treat `pat` as a dev tool rather than a dependency that can (should) be imported in detections (like `black`)

Reply:
Yeah, I think the only thing to do here is let CS know just in case. It may be that everybody's following the README, and they don't need to make any change if so 👍